client/src/components/controls/Controls.tsx

```diff
@@ -53,33 +38,23 @@ const Controls: React.FC<ControlsProps> = ({
   handleRemoveCourse,
 }) => {
   return (
-    <Grid container sx={{ paddingLeft: '66px' }} spacing={2}>
-      <Grid item container xs={12} md={6.5} direction="row">
-        <TermSelectWrapper>
-          <TermSelect />
-        </TermSelectWrapper>
-
-        <SelectWrapper minWidth={'296px'}>
-          <CourseSelect
-            assignedColors={assignedColors}
-            handleSelect={handleSelectCourse}
-            handleRemove={handleRemoveCourse}
-          />
-        </SelectWrapper>
-      </Grid>
-      <Grid item container direction="row" alignItems="center" justifyContent="space-between" xs={12} md={5.5}>
-        <CustomEventsWrapper>
-          <CustomEvents />
-        </CustomEventsWrapper>
-        <AutotimetablerWrapper>
-          <Autotimetabler handleSelectClass={handleSelectClass} />
-        </AutotimetablerWrapper>
-        <HistoryWrapper>
-          <History />
-        </HistoryWrapper>
-      </Grid>
-    </Grid>
+    <ControlsBar>
+      <TermSelectWrapper>
+        <TermSelect />
+      </TermSelectWrapper>
+      <SelectWrapper>
+        <CourseSelect
+          assignedColors={assignedColors}
+          handleSelect={handleSelectCourse}
+          handleRemove={handleRemoveCourse}
+        />
+      </SelectWrapper>
+      <ButtonRow>
+        <CustomEvents />
+        <Autotimetabler handleSelectClass={handleSelectClass} />
+      </ButtonRow>
+    </ControlsBar>
   );
 };
 
-export default Controls;
+export default Controls;
```

Comment:
add newline to end of file
```suggestion
export default Controls;

```